docs/index.md

```diff
@@ -1,32 +1,79 @@
 
-## Why Evidential?
+## What makes it different?
 
+- **Built for the social sector** – our roadmap is steered by a few mission-driven orgs, not by Fortune-500 advertisers  
+- **Free & open-source** – use our managed cloud in minutes or self-host for full control  
+- **Impact-first metrics** – it measures your users’ behaviour change and program outcomes, not vanity clicks  
+- **Complex multi-source programmes** – turn your various data sources into metrics once, then run experiments indefinitely  
+- **Hands-on support** – we guide you through hypothesis design, metric pipelines, launch and analysis, so you focus on learning what works, not on engineering
+
+## Key Features
+
+- Intuitive UI to create and monitor experiments  
+- A single dashboard centralizing all your lessons learnt  
+- Compatible with both your Data warehouse and Event engines  
+- Multiple methods: frequentist A/B, Bayesian, Multi-armed & Contextual bandits  
+- Comprehensive API to manage experiments programmatically  
+- *(Coming soon)* Integrations with the main messaging and survey platforms used by nonprofits
 
 ## What is it not?
 
+Evidential is not a product analytics tool, and it’s not built for full-scale Randomized Control Trials.
 
-## Who is it for?
+If you operate a single web or mobile app and want deep behavioural analytics, tools like Mixpanel or Amplitude are a better fit. These platforms assume a product and engineering team, and a substantial annual budget.
 
+Likewise, if you're planning a one-off RCT with full counterfactuals and enumerator-managed arms, you’ll need a different setup.
+
+Evidential shines when you want to continuously improve your complex program against its current version—10X faster than an RCT, but preserving its rigour.
 
 ## What kind of ideas can I test out?
 
+Here are some ideas our early partners have tested out:
+
+- Does the personalized AI response lead to better user engagement?  
+- Does using a casual tone with younger users lead to higher completion rates?  
+- Does reducing the number of questions asked during onboarding lead to lower drop-off rates?  
+- What is the best time to send messages to ensure users engage with it?  
+- Will this new feature lead to greater time spent on the platform?
+
+## Who are we?
+
+Evidential is a collaboration between [IDinsight](https://www.idinsight.org) and the [Agency Fund](https://www.agencyfund.org)—two teams that heard the same need: make experimentation painless for nonprofits.
+
+We combined complementary approaches into one shared platform ([read the blog](#)). Our team includes data scientists, engineers, and product designers with decades of experience building tools for both commercial and social sectors.
 
-## Design principles
+## Get started
 
+- [Read the quick-start guide](getting-started.md)  
+- Join the early-partner programme → email [support@evidential.dev](mailto:support@evidential.dev)  
```

Comment:
Where does this go? 
Does it get forwarded to both of us? support@evidential.dev got to